<launch>
  <node name="voice_mode" pkg="bobac2_audio" type="voice_mode_node" output="screen"/>
</launch>
